TombEngine/Game/misc.cpp

```diff
@@ -40,7 +40,7 @@ void TargetNearestEntity(ItemInfo* item, CreatureInfo* creature)
 	}
 }
 
-bool IsNextSectorValid(const ItemInfo& item, const Vector3& dir, float dist)
+bool IsNextSectorValid(const ItemInfo& item, const Vector3& dir, float dist, float canFloat)
```

Comment:
`canFloat` should be a bool.

Reply:
aah that was a bad mistake, yes it should be bool.  fixed^^